### PR TITLE
Style order

### PR DESCRIPTION
**Pull request by randy-darsh**
Related Issue: #104 

Summary: Added tables to the order index and show page

Author(s): randy-darsh and Joel "Big Dawg" Lindow

### CODE REVIEW

File: app/views/orders/index.html.erb [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
-		<%= order.created_at.to_formatted_s(:long_ordinal) %><br>
-		Status: <%= order.status %>
-	</li>
-	<%= link_to "View Order", order_path(order) %>
-<% end %>
-</ul>
-
+<table class="table">
+<thead>
+		<tr>
```
Comment (limsammy) [inline app/views/orders/index.html.erb]: Wow! This table looks great!

File: app/views/orders/show.html.erb [hunk 2]
```diff
@@ -5,16 +5,24 @@
...
+		<th>Quantity</th>
+		<th>Price</th>
+		<th>Subtotal</th>
+	</thead>
+	<tbody>
+		<% @items.each do |item| %>
+		<tr>
+			<td><%= link_to item.title, item_path(item) %></td>
+			<td><%= @order.item_quantity(item) %></td>
+			<td>$<%= @order.item_price(item) %> each</td>
```
Comment (limsammy) [inline app/views/orders/show.html.erb]: Nice catch with the dollar signs!

**Review by limsammy — APPROVED** [review]
Looks good to merge, will wait on other reviewers

**Review by JoelLindow — COMMENTED** [review]
I feel like the indentation could use a little bit of attention

File: app/views/orders/index.html.erb [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
+    <% current_user.orders.each do |order| %>
+      <tr>
+        <td><%= order.id %></td>
+        <td><%= order.created_at.to_formatted_s(:long_ordinal) %></td>
+        <td><%= order.status %></td>
+        <td><%= link_to "View Order", order_path(order) %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
```
Comment (amhursh) [inline app/views/orders/index.html.erb:21]: nice. much better in a table.

**Review by amhursh — APPROVED** [review]

**Pull request merged**